yt_dlp/extractor/indavideo.py

```diff
@@ -36,21 +35,36 @@ class IndavideoEmbedIE(InfoExtractor):
             'tags': ['tánc', 'cica', 'cuki', 'cukiajanlo', 'newsroom'],
         },
     }, {
-        'url': 'http://embed.indavideo.hu/player/video/1bdc3c6d80?autostart=1&hide=1',
+        'url': 'https://embed.indavideo.hu/player/video/3b63bc219c?autostart=1',
+        'md5': '8c82244ba85d2a2310275b318eb51eac',
+        'info_dict': {
+            'id': '1335611',
+            'ext': 'mp4',
+            'title': 'Vicces cica',
+            'description': 'Játszik a tablettel. :D',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'uploader': 'Jet_Pack',
+            'uploader_id': '491217',
+            'timestamp': 1390821212,
+            'upload_date': '20140127',
+            'duration': 7,
+            'age_limit': 0,
+            'tags': ['cica', 'Jet_Pack'],
+        },
+    }, {
```

Comment:
This is not what https://github.com/yt-dlp/yt-dlp/pull/8129/files#r1328046327 was asking for

Add a new class var, `_WEBPAGE_TESTS`, and add this test to it with its original URL (https://indavideo.hu/video/Vicces_cica_1)

We need to test embed extraction

Reply:
Sorry I missed it. Fixed in [this](https://github.com/yt-dlp/yt-dlp/pull/8129/commits/d57554f1acd6812f8878c981089106381476ddd7) commit.